src/test/scala/com/sksamuel/scapegoat/inspections/equality/ComparingUnrelatedTypesTest.scala

```diff
@@ -33,73 +32,6 @@ class ComparingUnrelatedTypesTest
         compileCodeSnippet(code)
         compiler.scapegoat.feedback.warnings.size shouldBe 1
       }
-      "for multiple comparisons" in {
```

Comment:
Why is that being removed?

Reply:
So this is dead code if you scroll further down.

Reply:
Right!